docs/requirements.txt

```diff
@@ -1,7 +0,0 @@
-lxml
```

Comment:
This file has to remain.  It is used by readthedocs.  This should have all docbuild requirement except doctr.

Reply:
thx. I didn't know RTD uses this. I will update